deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -157,8 +157,7 @@ async def create_recording_upload(
                     PULSE3D_UPLOADS_BUCKET,
                     upload_id=upload_id,
                 )
-
-                return UploadResponse(id=upload_id, params=params)
+                return UploadResponse(id=upload_id, params=params, usage_quota=usage_quota)
```

Comment:
Why does usage need to be returned from this route?

Reply:
I was anticipating this for when we add usage restrictions for uploads as well.

Reply:
Are we returning it in the route that creates a new job?

Reply:
Yes. the job route does return this so I think it would not make sense to keep it.